url.bs

```diff
@@ -3079,7 +3079,8 @@ is the <a for=/>origin</a> returned by running these steps, switching on <var>ur
    <li><p>Let <var>pathURL</var> be the result of <a lt="basic URL parser">parsing</a> the result of
    <a>URL path serializing</a> <var>url</var>.
 
-   <li><p>If <var>pathURL</var> is failure, then return a new <a>opaque origin</a>.
+   <li><p>If <var>pathURL</var> is failure, or <a>is not special</a> then return a new
+   <a>opaque origin</a>.
```

Comment:
This suggests that `blob:ws://example` would have a non-opaque origin. I think we should just support HTTP(S) URLs here.

Reply:
Ok, changed.